app/src/main/java/org/simple/clinic/drugs/selection/PrescribedDrugScreen.kt

```diff
@@ -77,9 +81,9 @@ class PrescribedDrugScreen(context: Context, attrs: AttributeSet) : LinearLayout
   private val delegate: MobiusDelegate<EditMedicinesModel, EditMedicinesEvent, EditMedicinesEffect> by unsafeLazy {
     MobiusDelegate.forView(
         events = events.ofType(),
-        defaultModel = EditMedicinesModel(),
+        defaultModel = EditMedicinesModel.create(patientUuid),
         update = EditMedicinesUpdate(),
-        effectHandler = EditMedicinesEffectHandler(this),
+        effectHandler = EditMedicinesEffectHandler.create(this, schedulersProvider),
```

Comment:
Should we move this to `AssistedInject` right now? Otherwise we'll keep having to manually provide dependencies that would otherwise be provided by DI.

Reply:
Yes. I can do that.